controllers/ovndbcluster_controller.go

```diff
@@ -786,6 +806,69 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 	return ctrl.Result{}, nil
 }
 
+// generateServiceConfigMaps - create create configmaps which hold service configuration
+func (r *OVNDBClusterReconciler) generateExternalConfigMaps(
+	ctx context.Context,
+	h *helper.Helper,
+	instance *ovnv1.OVNDBCluster,
+	serviceName string,
+	envVars *map[string]env.Setter,
+) error {
+	// Create/update configmaps from templates
+	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(serviceName), map[string]string{})
+	encapType := ""
+
+	externalEndpoint, err := instance.GetExternalEndpoint()
+	if err != nil {
+		return err
+	}
+
+	ovnController, err := ovnv1.GetOVNController(ctx, h, instance.Namespace)
```

Comment:
nit: it may simplify the code a bit if you can drop `encapType` variable like:

```
externalTemplateParameters := make(map[string]interface{})
externalTemplateParameters["OVNRemote"] = externalEndpoint

ovnController, ... = ...GetOVNController(...)
if err != nil {
    return err
}
if ovnController != nil {
    externalTemplateParameters["OVNEncapType"] = ovnController.Spec.ExternalIDS.OvnEncapType
}
```

Reply:
Oh, thanks! Done